application/common/components/Emailer.php

```diff
@@ -24,7 +60,7 @@ class Emailer extends Component
      * @param string $subject The subject.
      * @param string $htmlBody The email body (as an HTML string).
      */
-    public function email(string $toAddress, string $subject, string $htmlBody)
+    protected function email(string $toAddress, string $subject, string $htmlBody)
```

Comment:
The [strip_tags](http://php.net/manual/en/function.strip-tags.php) function isn't perfect for converting HTML to text, but for our emails it will probably cover 98%+ of what we'll need to do (missing things like `&quot;` and other HTML entities).